CHANGELOG.md

```diff
@@ -26,7 +26,8 @@ at anytime.
   * `get` failing with a non-useful error message when given a uri for a channel claim
   * exception checking in several wallet unit tests
   * daemon not erring properly for non-numeric values being passed to the `bid` parameter for the `publish` method
-  *
+  * publish command to allow updating claims with bid amount higher than wallet balance(by spending the claimtrietx coin) (https://github.com/lbryio/lbry/issues/748)
```

Comment:
Commands, classes, functions, and argument names should be decorated with code ( ``` ` ```) tags. In this entry, `publish` and `bid` should be decorated.

Reply:
>higher than wallet balance

How much higher? Could be rephrased: `with a bid amount higher than the wallet balance, so long as the amount is less than the wallet balance plus the bid amount of the claim being updated`